clicktests/helpers.js

```diff
@@ -105,3 +107,7 @@ helpers.selectAllText = function(page) {
   helpers.log('Trying to select all in focused element (ctrl-A)');
   page.sendEvent('keypress', page.event.key.A, null, null, 0x04000000 );
 }
+helpers.getPort = function() {
```

Comment:
That is very similar to what I have tried: https://gist.github.com/mikeal/1840641, which I liked it better because it's less dependancy.   However within phantomjs client code, `net` package is unavailable.  phantom has [webserver](http://phantomjs.org/api/webserver/) module but its has "experimental" tag.  

Thus I've went with try and retry with random port.